ccd-definition/CaseEvent.json

```diff
@@ -425,5 +425,31 @@
     "ShowSummary": "N",
     "ShowEventNotes": "Y",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
As mentioned earlier please remove migrateCase

Reply:
👍

Reply:
I see it's removed now.